deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Net461.wxs

```diff
@@ -535,6 +535,11 @@
               Source="$(var.TracerHomeDirectory)\net461\System.Xml.XPath.XDocument.dll"
               KeyPath="yes" Checksum="yes"/>
       </Component>
+      <Component Win64="$(var.Win64)">
```

Comment:
Do we really want the installer to copy the benchmark exporter? I feel like it only makes sense through a nuget package

Reply:
Yeah, make sense in a nuget package, it's probably better to have something like `Datadog.Trace.BenchmarkDotNet` depending on `Datadog.Trace`

Reply:
wix configuration files has been restored (https://github.com/DataDog/dd-trace-dotnet/pull/817/commits/f2ac8cb45f84657d3bac1042ecf624ae23413bb4)